content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -729,25 +729,25 @@ $ kubectl get pods -n foo -o name -l istio.io/gateway-name=httpbin-gateway | sed
 
 * Remove the authorization policy:
 
-{{< tabset category-name="config-api" >}}
+    {{< tabset category-name="config-api" >}}
```

Comment:
I'm not sure we need the additional indentation in this section: 
https://deploy-preview-13230--preliminary-istio.netlify.app/latest/docs/tasks/security/authorization/authz-ingress/#clean-up
vs
https://preliminary.istio.io/latest/docs/tasks/security/authorization/authz-ingress/#clean-up

Reply:
The original layout looks better. This section is reverted 😸